pennylane/drawer/tape_text.py

```diff
@@ -556,7 +588,7 @@ def tape_text(
         if i == final_operations_layer:
             wire_totals = [f"{s}─┤" for s in wire_totals]
             for b in range(n_bits):
-                if cwire_layers[b][-1] > final_operations_layer:
+                if cwire_layers[b][-1][-1] > final_operations_layer:
```

Comment:
`cwire_layers[b]` is now a list of lists, and we want the last layer index of the last usage of the cwire `b`